core/server/common/src/main/java/alluxio/master/MasterProcess.java

```diff
@@ -175,10 +175,14 @@ public boolean waitForReady(int timeoutMs) {
   }
 
   protected void startRejectingServers() {
-    mRejectingRpcServer = new RejectingServer(mRpcBindAddress.getPort());
-    mRejectingRpcServer.start();
-    mRejectingWebServer = new RejectingServer(mWebBindAddress.getPort());
-    mRejectingWebServer.start();
+    if (mRejectingRpcServer == null) {
```

Comment:
If the original rejecting server host the port, creating a new rejecting server with the same port will not fail or show errors?

Reply:
Judging by the current code for `RejectingServer` it'll fail and quit without us knowing it.